frontend/src/App.js

```diff
@@ -1,7 +1,47 @@
+/** @jsx jsx */
+import emotionReset from "emotion-reset";
+import { jsx, Global, css } from "@emotion/core";
 import React from "react";
+import FlightDetails from "./views/flight-details";
 
 function App() {
-  return <div>hi</div>;
+  const cssWrapper = {
+    backgroundColor: "lightgray",
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center"
+  };
+
+  return (
+    <>
+      <Global
+        styles={css`
+          ${emotionReset};
+          body {
+            font-family: "Roboto", sans-serif;
+            line-height: 1.5;
+          }
+        `}
+      />
+      <div css={cssWrapper}>
+        <div
+          css={{
+            maxWidth: 678,
```

Comment:
If you start writing css in a way (literal css), then I'd recommend to stick to it.

Reply:
Yes, you right, but the library that I'm using is written in plain css. I'm going to look for another options.
 